app/components/atoms/social-media-buttons/social-media-button.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+
+import {
+  InstagramButton,
+  FacebookButton,
+  TwitterButton,
+} from './social-media-button.component';
+
+export default {
+  title: 'Atoms/SocialMediaButton',
+} as ComponentMeta<
+  typeof InstagramButton | typeof FacebookButton | typeof TwitterButton
+>;
+
+const Template: ComponentStory<
+  typeof InstagramButton | typeof FacebookButton | typeof TwitterButton
+> = (args) => (
+  <>
+    <div
+      style={{display: 'flex', justifyContent: 'space-evenly', width: '20em'}}
+    >
+      <InstagramButton {...args} />
```

Comment:
why do we have a style?, can you please just make a different component per each of those, see: https://github.com/tepachelabs/perro-cafe-storefront/pull/33/files